README.md

```diff
@@ -33,16 +33,67 @@ You can set the environment variable `FASTLANE_OPT_OUT_USAGE` to opt out.
 
 Alternatively, add `opt_out_usage` to your `Fastfile`.
 
+# Running _enhancer_ locally
+
+* Start with a `bundle install` in the project directory to make sure you have all of the dependencies
+
+## Set needed environment variables
+
+* Create a `.env` file in your _enhancer_ project root containing the desired values for:
+
+```
+ANALYTIC_INGESTER_URL=[the public URL for Analytic Ingester]
+FL_PASSWORD=[your desired password]
```

Comment:
It might not be clear what password, maybe phrase it as auth password to see the graphs